apps/web/src/server/applications/create/applicant/__tests__/applications-create-applicant.test.js

```diff
@@ -123,4 +123,51 @@ describe('applications create applicant', () => {
 			});
 		});
 	});
+
+	describe('GET /applicant-email', () => {
```

Comment:
can you nest the GET/ and the POST/ inside a 

```
describe('Applicant email', () => { 
    describe(GET... etc...)
     describe(POST...etc...)
})
```

Reply:
also, very small detail: baseUrl is used by both POST and GET. You can move it up in the global scope once you create it